pkg/tsdb/testdatasource/stream_handler.go

```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"fmt"
 	"math/rand"
+	"regexp"
 	"strings"
 	"time"
 
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
 )
 
+var random20HzStreamRegex = regexp.MustCompile(`random-20Hz-stream(-\d+)?`)
```

Comment:
Are these changes related in some way to main PR concept (i.e. modifying barchart to show all values in tooltip?) I'm good with including PORK as part of PRs but it may be nice to call out what PORK exists inside PR description (assuming these changes are intended 🙂)

Reply:
ha -- nope -- it was based on merging into a not yet merged PR, but the PR ended up including all the recently committed values in main :man_shrugging: